rrfs-test/validated_yamls/templates/obtype_config/adpupa_specificHumidity_120.yaml

```diff
@@ -0,0 +1,222 @@
+
+       obs filters:
+         # ------------------
+         # specificHumidity (120)
+         # ------------------
+         # Reject all obs with QualityMarker > 3
+         - filter: RejectList
+           apply at iterations: 0,1
+           where:
+           - variable: ObsType/specificHumidity
+             is_in: 120
+           - variable: QualityMarker/specificHumidity
+             is_in: 4-15
+
+         # Time window filter
+         - filter: Domain Check
+           apply at iterations: 0,1
+           where:
+             - variable:
+                 name: MetaData/timeOffset # units: s
+               minvalue: -5400
+               maxvalue:  5400
+
+         # Online domain check
+         - filter: Bounds Check
+           filter variables:
+           - name: specificHumidity
+           test variables:
+           - name:  GeoVaLs/observable_domain_mask
+           minvalue: 0.0
+           maxvalue: 0.5
+
+         # Initial error assignment
+         - filter: Perform Action
+           filter variables:
+           - name: specificHumidity
+           where:
+           - variable: ObsType/specificHumidity
+             is_in: 120
+           action:
+             name: assign error
+             error function:
+               name: ObsFunction/ObsErrorModelStepwiseLinear
+               options:
+                 xvar:
+                   name: MetaData/pressure
+                 xvals: [110000, 105000, 100000, 95000, 90000, 85000, 80000, 75000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000, 500, 400, 300, 200, 100, 0]
+                 errors: [0.056103, 0.063026, 0.073388, 0.086305, 0.099672, 0.1121, 0.12347, 0.13337, 0.14214, 0.15031, 0.15641, 0.1594, 0.15962, 0.16026, 0.16419, 0.17154, 0.1798, 0.18754, 0.19339, 0.19701, 0.19887, 0.19966, 0.19993, 0.2, 0.2, 0.2, 0.19999, 0.19999, 0.19999, 0.20001, 0.20004, 0.19999, 0.19949]
+
+         # Error inflation based on pressure check (setupq.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: specificHumidity
+           where:
+           - variable: ObsType/specificHumidity
+             is_in: 120
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: specificHumidity
+                 inflation factor: 8.0
+                 request_saturation_specific_humidity_geovals: true
+
+         # Error inflation based on errormod (qcmod.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: specificHumidity
+           where:
+           - variable: ObsType/specificHumidity
+             is_in: 120
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorConventional
```

Comment:
Do you know if this filter is needed for mesonet obs too? I think the `errormod` function is called for all obs in `read_prepbufr.f90`. But you've shown that the obs errors match really well for mesonet so it might not be needed.

Reply:
I was wondering the same thing. I plan to do some extra tests with it in my next round of testing in phase 3.

Reply:
I tested this and the final obserrors looked obviously different compared to GSI. I don't think it is needed.